CMakeLists.txt

```diff
@@ -110,7 +110,7 @@ include(CMakeDependentOption)
 # TODO(DF): We may end up needing to support building of both shared
 #   and static libs simultaneously. Also, using this means we inherit
 #   the option if openassetio is used as a CMake subproject.
-option(BUILD_SHARED_LIBS "Set to OFF to build static libraries" ON)
+option(BUILD_SHARED_LIBS "Set to OFF to build static libraries. Not recommended." ON)
```

Comment:
I know what you mean here, but would another reader think 'Not recommended' is referring to 'don't use this option'?

Should it be clarified to 'Static libraries are not recommended'?

This would at least be consistent with the warning message that follows below.

Reply:
Good point, rather ambiguous as it stands.

Reply:
Updated in 9f4a1037